### ISSUE TITLE
Add flag poles

### ISSUE THREAD

**Issue by Shlomoraz9983** [krikienoid/flagwaver#71]
can you add different types of flagpoles?

**Comment by krikienoid** [comment]
What do you mean? There is an option for a horizontal flagpole, a crossbar flagpole and a couple others

**Comment by Shlomoraz9983** [comment]
no like diffrent types of flagpoles 
like golden, wood, etc...

**Comment by Shlomoraz9983** [comment]
that you will add diffrent colors/more flagpoles

**Comment by Shlomoraz9983** [comment]
![BROTHER](https://user-images.githubusercontent.com/116600899/197757796-47dbd590-99d9-4412-887d-895002089f62.jpg)
like this one with a spike

**Comment by Shlomoraz9983** [comment]
or a one with a gold circle one on top
![image](https://user-images.githubusercontent.com/116600899/197760169-85d8c625-02b8-496a-92af-7acf7ebd3ac8.png)

**Comment by Shlomoraz9983** [comment]
basically more stuff on top and colors

**Comment by Shlomoraz9983** [comment]
![image](https://user-images.githubusercontent.com/116600899/197762582-41220acf-b35f-43ce-9cc3-49f1c25fa9b4.png)
or add this flagpole

**Comment by Shlomoraz9983** [comment]
What do you think?

**Comment by Shlomoraz9983** [comment]
good idea?

**Comment by krikienoid** [comment]
I don't think I would be able to do the eagle without a 3d model to work off of. The ball and the spike are probably do-able, I don't have much experience with creating my own 3d models, but simple shapes like balls and cones can be generated with code.

**Comment by Shlomoraz9983** [comment]
so maybe search for a 3d model of a spike?

**Comment by Shlomoraz9983** [comment]
yeah we dont need the eagle but we need also diffrent colored poles

**Comment by Shlomoraz9983** [comment]
like gold ones

**Comment by Shlomoraz9983** [comment]
and we need does gold things around the flag

**Comment by JuanLovesUruguay** [comment]
![D_NQ_NP_634324-MLM31224938681_062019-O](https://user-images.githubusercontent.com/118695472/202929983-39ea2792-41db-4979-920a-32e9abc99785.jpg)

**Comment by Shlomoraz9983** [comment]
yeah like this
